corehq/apps/callcenter/tasks.py

```diff
@@ -58,12 +71,18 @@ def sync_usercases_if_applicable(user, spawn_task):
         return
     if (user.project.call_center_config.enabled or user.project.usercase_enabled):
         if spawn_task:
-            sync_usercases_task.delay(user._id)
+            sync_usercases_task.delay(user._id, user.domain)
         else:
-            sync_usercases_task(user._id)
+            sync_usercases(user, user.domain)
+
+
+@task(queue='background_queue')
+def sync_web_user_usercases_task(user_id, domain):
+    user = CouchUser.get_by_user_id(user_id)
+    sync_usercases(user, domain)
 
 
-@task(serializer='pickle', queue='background_queue')
-def sync_usercases_task(user_id):
+@task(queue='background_queue')
+def sync_usercases_task(user_id, domain):
     user = CommCareUser.get_by_user_id(user_id)
-    sync_usercases(user)
+    sync_usercases(user, domain)
```

Comment:
These could be combined into a single tasks since `CouchUser.get_by_user_id` works for both web and mobile users
```python
@task(queue='background_queue')
def sync_usercases_task(user_id, domain):
    user = CouchUser.get_by_user_id(user_id)
    sync_usercases(user, domain)
```